src/docs/clients/cocoa/migration.mdx

```diff
@@ -13,23 +13,23 @@ tags: []
 
 Update your `Podfile` to include `:subspecs => ['Core', 'KSCrash']`:
 
-  ```ruby
-  source 'https://github.com/CocoaPods/Specs.git'
-  platform :ios, '8.0'
-  use_frameworks!
+```ruby
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '8.0'
+use_frameworks!
 
-  target 'YourApp' do
-      pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :subspecs => ['Core', 'KSCrash'], :tag => '3.x.x'
-  end
-  ```
+target 'YourApp' do
+    pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :subspecs => ['Core', 'KSCrash'], :tag => '3.x.x'
```

Comment:
Here is uses 4-space indentation

Reply:
Drat! How did I miss that? I swear I looked at all of them. (It's not prettier - those I had to fix by hand after you pointed them out.)

Maybe it got lost in all the rebasing. Anyway, will fix again - thanks for catching!

Reply:
If another run of `prettier` would undo the manual tweaks, then it might not be worth tweaking, just going with whatever comes out of prettier.

Reply:
No, it's not that it undoes it. Prettier doesn't touch this at all. I figured out what happened, though. I have my find in VSCode set to exclude stuff from the old `*/clients/*` docs, so I didn't catch those the first time through.